.github/workflows/zfs-qemu.yml

```diff
@@ -49,8 +49,8 @@ jobs:
         # rhl:     almalinux8, almalinux9, centos-stream9, fedora40, fedora41
         # debian:  debian11, debian12, ubuntu20, ubuntu22, ubuntu24
         # misc:    archlinux, tumbleweed
-        # FreeBSD Release: freebsd13-3r, freebsd13-4r, freebsd14-0r, freebsd14-1r
-        # FreeBSD Stable:  freebsd13-4s, freebsd14-1s
+        # FreeBSD Release: freebsd13-3r, freebsd13-4r, freebsd14-0r, freebsd14-1r, freebsd14-2r
+        # FreeBSD Stable:  freebsd13-4s, freebsd14-1s, freebsd14-2s
```

Comment:
We don't need 14.1-STABLE any more. Actually in FreeBSD both are just called 14-STABLE.  Not sure why the image name is more precise, may be just to vaguely specify time frame.

Reply:
Removed.